Cargo.toml

```diff
@@ -10,6 +10,7 @@ resolver = "2"
 heap-size = 134217728
 stack-size = 2097152
 threads = 6
+debug = false
```

Comment:
:information_source: we'll later enable cipher to use the key manager for confidential information, so we're turning off debug mode.